index.html

```diff
@@ -1644,6 +1644,141 @@ <h4>Background</h4>
 
 </section> <!-- Alice Rents a car -->
 
+	    <section id="messaging">
+      <h3>Portable Secure Communication</h3>
+      <section id="messagingBackground">
+        <h4>Background</h4>
+        <p>
+          Designing systems that securely support the scale of global communication on the Web has traditionally
+		come with a distinct set of tradeoffs that often negatively impact end-users. Current
+		<abbr title="Public Key Infrastructure">PKI</abbr> systems tightly control who gets to manage
+		and control the cryptographic keys associated with the set of digital certificates that serve as
+		the backbone of the internet. This constraint means that modern cryptography is largely unusable for
+		the average user, forcing them to borrow or ‘rent’ identifiers such as email addresses, usernames,
+		and website domains through systems like <a href="https://en.wikipedia.org/wiki/Domain_Name_System">DNS</a>,
+		<a href="https://en.wikipedia.org/wiki/X.509">X.509</a>, and <a href="https://en.wikipedia.org/wiki/Social_network">social
+		networks</a>. By not directly controlling their own identifiers, people who want to use secure
+		communication are not only unable to utilize cryptographic security in many of their everyday
+		interactions, they are also limited to choosing between traditional secure messaging services
+		to safely communicate with friends and family in a limited, chat-style context.
+        </p>
+        <p>
+          In a traditional secure messaging approach the security is a property of the platform; if a user changes
+		platforms, either temporarily or permanently, they get their security a different way, tied to a
+		different login and different terms of service. DIDs take security out of the hands of the
+		context owner, i.e. an application or service provider, and put it in the hands of the DID
+		controller who can choose which contexts they want to communicate in and when. This makes
+		security decentralized and portable, instead of having security of an interaction tied to
+		the protocol, transport, or application used to facilitate it. This feature allows people
+		to easily switch communication contexts without losing any security guarantees, and provides
+		a resilient backbone for all internet communications. Moving away from traditional identifiers
+		such as phone numbers and email addresses can help to break down the siloed ecosystems that
+		prevent data portability and make it difficult to implement secure communication technology
+		more broadly into applications and services. Current approaches to secure communication
+		often have the following drawbacks:
```

Comment:
```suggestion
In a traditional secure messaging approach the security is a property of the platform; if a user changes
		platforms, either temporarily or permanently, they get their security a different way, tied to a
		different login and different terms of service. DIDs take security out of the hands of the
		context owner, i.e., an application or service provider, and put it in the hands of the DID
		controller, who can choose the contexts in which they want to communicate and when. This makes
		security decentralized and portable, instead of having the security of an interaction tied to
		the protocol, transport, or application used to facilitate it. This feature allows people
		to easily switch communication contexts without losing any security guarantees, and provides
		a resilient backbone for all internet communications. Moving away from traditional identifiers
		such as phone numbers and email addresses can help to break down the siloed ecosystems that
		prevent data portability and make it difficult to more broadly incorporate secure communication 
		technology in applications and services. Current approaches to secure communication
		often have drawbacks such as the following:
```